libraryBuilds/gson/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // This is the build output of the plugin itself.
+        classpath fileTree(dir: '../../j2objc-gradle/build/libs', include: ['*.jar'])
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'com.github.j2objccontrib.j2objcgradle'
```

Comment:
Of course. Makes sense.

On Sat, Sep 26, 2015 at 8:58 PM Advay Mengle <notifications@github.com>
wrote:

> In libraryBuilds/gson/build.gradle
> <https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/pull/23#discussion_r40501130>
> :
>
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +buildscript {
> > +    repositories {
> > +        jcenter()
> > +    }
> > +    dependencies {
> > +        // This is the build output of the plugin itself.
> > +        classpath fileTree(dir: '../../j2objc-gradle/build/libs', include: ['*.jar'])
> > +    }
> > +}
> > +
> > +apply plugin: 'java'
> > +apply plugin: 'com.github.j2objccontrib.j2objcgradle'
>
> We need to be able to dynamically vary the classpath to be either the
> maven repo at plugins.gradle.org OR the local build directory (to
> eventually achieve #33
> <https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/issues/33>
> ).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/pull/23/files#r40501130>
> .
>